### PR TITLE
T15413 Fixed raw value bug in lowupdate

### LINKED ISSUE #15413: [BUG]: Dynamic update model issue
**Describe the bug**
Same as [#12773](https://github.com/phalcon/cphalcon/issues/12773),  when field type is `Decimal` or `Float`。

If old value in database is 1.0, then use RawValue to update this field, will reproduce this bug. Because of following logic
```
case Column::TYPE_DECIMAL:
case Column::TYPE_FLOAT:
    let changed = floatval(snapshotValue) !== floatval(value);
    break;
```

Because:
```
floatval(new \Phalcon\Db\RawValue("f+1")) === float(1)
```

So to reproduce the issue, should be 

1. snapshotValue is 1, 
2. use  RawValue to update this field
3. the field type is DECIMAL or FLOAT

### PR DESCRIPTION
**Pull request by BeMySlaveDarlin**
Hello!

* Type: bug fix
* Closes #15413

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

Thanks

### CODE REVIEW

File: phalcon/Mvc/Model.zep [hunk 1]
```diff
@@ -3900,6 +3900,16 @@ abstract class Model extends AbstractInjectionAware implements EntityInterface,
                                         );
                                     }
 
+                                    /**
+                                     * Get actual values before comparison
+                                     */
+                                    if is_object(snapshotValue) && snapshotValue instanceof RawValue {
```
Comment (Jeckerson) [inline phalcon/Mvc/Model.zep:3906]:
```suggestion
                                    if snapshotValue instanceof RawValue {
```

File: phalcon/Mvc/Model.zep [hunk 2]
```diff
@@ -3900,6 +3900,16 @@ abstract class Model extends AbstractInjectionAware implements EntityInterface,
...
+                                        let snapshotValue = snapshotValue->getValue();
+                                    }
+                                    if is_object(value) && value instanceof RawValue {
```
Comment (Jeckerson) [inline phalcon/Mvc/Model.zep:3909]:
```suggestion
                                    if value instanceof RawValue {
```

File: phalcon/Mvc/Model.zep (hunk 1 above)
Comment (BeMySlaveDarlin) [reply phalcon/Mvc/Model.zep:3906]: `instanceof` expects an object instance, but `snapshotValue` can be other type

File: phalcon/Mvc/Model.zep (hunk 2 above)
Comment (BeMySlaveDarlin) [reply phalcon/Mvc/Model.zep:3909]: `instanceof` expects an object instance, but `value` can be other type

**Review by niden — APPROVED** [review]

**Pull request merged**